Client/src/Pages/Login/index.css

```diff
@@ -18,7 +18,9 @@
 }
 
 .login-form-header-logo {
-  margin: 0 auto;
+  margin: 30px auto;
+  margin-bottom: 60px;
+  width: 353px;
```

Comment:
I don't believe there is a need for `auto` here. I'm not so sure about the margins either, we're only really concerned about spacing the logo and the login header apart so a margin-bottom should suffice.

Reply:
In fact, auto is what's helping to make the Logo be centered.
If I remove it, logo will be right aligned!

Reply:
I added margins to make it a bit more clear and clean, since the "Log in to your account" heading is too large and it wouldn't look too nice without these margins.
I suggest keeping it for now. If we did not like it, we'll remove it later.

Reply:
I've tested as well, the auto isn't required as the parent has text-align center set on it